nni/algorithms/hpo/hyperband_advisor/test/config_hyperband.yml

```diff
@@ -0,0 +1,27 @@
+authorName: default
```

Comment:
@liuzhe-lz is this a correct place to put tests?

Reply:
I will check it if this can move to UT.

Reply:
This test need nasbench201, and installing nasbench201 will cost 2h in my machine, so maybe temporarily put it here?